deployments/aws/templates/stack-services.yaml

```diff
@@ -106,6 +106,16 @@ Parameters:
     Description: |
       The name of the subdomain that will be created under ParentDomain for the service
     Default: keycloak
+  RoleName:
+    Type: String
+    Description: |
+      The name of the AIM role that will be created to facillitate logging.
+  InstanceProfileName:
+    Type: String
+    Description: |
+      The name of the AIM role that will be created to facillitate logging.
```

Comment:
Is this AIM or IAM role, as above?

Reply:
it is IAM (Identy and Access Management). Thanks, will fix